src/docs/frontend/index.mdx

```diff
@@ -151,16 +151,14 @@ We are currently exploring alternatives to the `Reflux` library for future use.
 ## Testing
 
 <Alert level="info">
-  We are moving away from Enzyme in favor of React Testing Library. For RTL tips check out <Link to="/frontend/using-rtl/">this page</Link>.
+  For React Testing Library (RTL) tips check out <Link to="/frontend/using-rtl/">this page</Link>.
 </Alert>
 
-Note: Your filename needs to be .spec.jsx or jest won’t run it!
+Note: Your filename needs to be `.spec.jsx` or `.spec.tsx` for jest to run run it!
```

Comment:
let's get rid of even suggesting `jsx`?

Reply:
there are still a handful of jsx files left. Lets fix them up first then we can fix this to match.